Sources/_FoundationInternals/String/String+Internals.swift

```diff
@@ -54,3 +54,12 @@ extension String {
     }
 
 }
+
+extension DefaultStringInterpolation {
```

Comment:
I am not really happy with this. I think we can do better here. Are we planning to add the `String(format)` init in this package?

Reply:
Ultimately we will have access to all the interpolation-based formatters, so this seems like an appropriate approach.